### PR TITLE
Add CO'24 Grad Issue visuals

### PR DESCRIPTION
**Pull request by tiffanyl9**
Add Grad Issue class of 2024 charts.

## What does this PR do?
Add Grad Issue Class of 2024 pie (employment by industry) and bar (employment vs. continuing education) charts.

Also add map!

Screenshot 2024-04-26 at 2 27 35 PM

## How was the functionality tested and verified?
All steps should be completed in the order presented, and you should not move on to the next step until the previous is completed.
- [x] The title of the `html` page is the same as the chart.
- [x] The `meta_data.json` file is filled in correctly (the description is a complete sentence, the title is the same as the visual).
- [x] The chart and its associated pages have been minified using the `npm run process-visual` command.
- [x] The visual is styled similar to other charts (centered title/subtitle, no legends unless absolutely necessary, etc.)
- [ ] This visual was approved by the lead developer.
- [x] The visual was approved by the story's stakeholders.

### CODE REVIEW

File: 2024/classof2024_ed_vs_empl/main.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        align: 'left'
+    },
+
+    subtitle: {
+        text: 'The ratio of students pursuing employment versus continuing their education has remained roughly constant from the Class of 2004 to the Class of 2024. The "Continuing Education" category includes students pursuing professional degrees, PhDs, and academic masters.',
+        align: 'left',
+        style: {
+            fontSize: '16px'
+        }
+    },
```
Comment (michplunkett) [inline 2024/classof2024_ed_vs_empl/main.js]: The title and subtitle should be centered.

File: 2024/classof2024_tl/main.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    },
+    title: {
+        text: 'Employment by Industry'
+    },
+    tooltip: {
+        valueSuffix: '%'
+    },
+    subtitle: {
+        text: 'Source:<a href="https://www.mdpi.com/2072-6643/11/3/684/htm" target="_default">MDPI</a>'
+    },
```
Comment (michplunkett) [inline 2024/classof2024_tl/main.js]: Same with these two.

File: 2024/classof2024_tl/index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        <link rel="stylesheet" href="../../static/main.css" />
+        <!-- /build -->
+    </head>
+
+    <body>
+        <div id="chart-div"></div>
+
+        <!-- EXTERNAL JS FILES -->
+        <!-- YOU LIKELY DO NOT NEED ALL OF THESE -->
+        <!-- IF YOU AREN'T USING AN IMPORT, IT'S RECOMMENDED THAT YOU REMOVE IT -->
```
Comment (michplunkett) [inline 2024/classof2024_tl/index.html:15]: Do you need all of these imports? Same for the other visual.

**Comment by michplunkett** [comment]
Update the title of the pull-request when you can to something like `Add class of 2024 visuals`. It helps makes the branch history more sensical.

Screenshot 2024-04-26 at 12 18 07 AM

**Comment by Nik4002** [comment]
A few things:
1. About the bar chart, if you think my comment about flipping the colors makes sense, then do that
2. Don't forget to minify the files when you're done

Otherwise, looks good to me!

**Review by michplunkett — CHANGES_REQUESTED** [review]
Specifically the title things and removing any unused highcharts modules.

**Comment by Nik4002** [comment]
Just to clarify, @asteinhart, we decided that you will just screenshot the map and publish it that way? Also, about the subtitle, I feel like that sentence might be better in the plot caption.

**Review by michplunkett — APPROVED** [review]

**Pull request merged**